zipkin-storage/elasticsearch/src/main/resources/zipkin/storage/elasticsearch/zipkin_template.json

```diff
@@ -12,6 +12,16 @@
           "filter": [
             "lowercase"
           ]
+        },
+        "traceId_analyzer": {
```

Comment:
@openzipkin/elasticsearch My assumption is that this is not a costly step, and it also doesn't seem to increase the size of index needed (since this doesn't create multiple tokens). If these assumptions are invalid, report back!